src/Cli.Tests/ConfigGeneratorTests.cs

```diff
@@ -177,8 +178,8 @@ public void TestSpecialCharactersInConnectionString()
             }");
 
         expectedRuntimeConfigJson = expectedRuntimeConfigJson.Replace(" ", string.Empty);
-        expectedRuntimeConfigJson = expectedRuntimeConfigJson.Replace("\n", string.Empty);
         expectedRuntimeConfigJson = expectedRuntimeConfigJson.Replace("\r\n", string.Empty);
+        expectedRuntimeConfigJson = expectedRuntimeConfigJson.Replace("\n", string.Empty);
```

Comment:
Changed the order for consistency between replacements for actual and expected configs. Since the order was different, the replacements would have been different as well. This prevents any unexpected failures.